docs/install/uninstallEverest.md

```diff
@@ -105,8 +107,12 @@ Below is a list of CRDs for different operators.
     ??? example "PostgreSQL operator (Percona and CrunchyData)"
         ```
         perconapgbackups.pgv2.percona.com
-        perconapgrestores.pgv2.percona.com
         perconapgclusters.pgv2.percona.com
+        perconapgrestores.pgv2.percona.com
+        perconapgupgrades.pgv2.percona.com
+        crunchybridgeclusters.postgres-operator.crunchydata.com
+        pgadmins.postgres-operator.crunchydata.com
+        pgupgrades.postgres-operator.crunchydata.com
```

Comment:
@rasika-chivate I think you already have another PR with these changes, beware of conflicts...

Reply:
These have already been updated. Will reject these changes.